recipes/qh3/conda_build_config.yaml

```
@@ -0,0 +1,2 @@
+c_stdlib_version:       # [osx and x86]
+  - '10.12'             # [osx and x86]
```

```suggestion
  - '10.13'             # [osx and x86]
```
10.13 is now the new default/minimum